### ISSUE TITLE
Ok to have 2 styles in a .teat.ts file?

### ISSUE THREAD

**Issue by phil-lgr** [seek-oss/treat#68]
Treat file with 2 styles:

![image](https://user-images.githubusercontent.com/9644867/68077258-06f32d80-fd97-11e9-816d-f37cd5cb8bff.png)

Usage in component: 

![image](https://user-images.githubusercontent.com/9644867/68077267-28541980-fd97-11e9-809e-c3db949514cc.png)

I would expect that 2 classes are generated, for `nav` and for `logo`, but I end up with:

![image](https://user-images.githubusercontent.com/9644867/68077275-4e79b980-fd97-11e9-9b52-ad56f7f26a19.png)

basically the `nav` styles get overwritten by the `logo` style is this expected?

**Comment by phil-lgr** [comment]
I can get over this using the debugLocalName prop:

```ts
export const links = style(
  () => ({
    display: 'flex',
    alignItems: 'center'
  }),
  'links'
);
```

**Comment by mattsjones** [comment]
This isn't expected behaviour. Would be happy to have a look at a small repo reproducing the issue. 

Btw you don't need to use the theme callback style if you're not accessing the theme variables.

**Comment by phil-lgr** [comment]
Sounds good, I'll find some time to investigate some more, I'm leaving this open in the meantime, I'll update the issue sometimes this week

**Comment by phil-lgr** [comment]
I'll close this for now and will reopen if it arises again
